lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -479,6 +480,8 @@ def filter_query(
                 # columns in the Dataset table.
                 if second == Metadata.METALOG:
                     native_key = keys.pop(0).lower()
+                elif second == "owner":
+                    filter = make_operator(User.username, term.operator, value)
```

Comment:
`metalog` has a `Metadata` constant because it's actually a primary key value in the metadata table. `owner` isn't, and the dataset/metadata code doesn't need it so it doesn't really belong in that file. Yeah, there are a couple of uses that could in theory have been consolidated somewhere, but there isn't really an obvious home and it didn't seem worthwhile.